common/setups/rasr/util/gmm.py

```diff
@@ -233,6 +235,26 @@ def __init__(
         self.sdm_args = sdm_args
 
 
+@dataclass
+class PrevCtm:
+    """
+    Defines a previous ctm output from a recognition which is used to extract the
+    speaker alignments for SAT adaptation for the dev/train sets.
+
+    :param prev_step_key: name of the step, e.g. "tri"
+    :param pronunciation_scale: one of the existing pronunciation scales
+    :param lm_scale: one of the existing lm-scales
+    :param iteration: one of the existing GMM iterations
+    :param use optimized recognition
```

Comment:
```suggestion
    :param optimized_lm: use optimized recognition
```
missing parameter name

```suggestion
    :param optimized_lm: use automatically optimized lm.scale which might differ from param lm_scale 
```
better docstring suggestion